client/gui/Shortcut.h

```diff
@@ -65,6 +65,10 @@ enum class EShortcut
 	MAIN_MENU_CAMPAIGN_ROE,
 	MAIN_MENU_CAMPAIGN_AB,
 	MAIN_MENU_CAMPAIGN_CUSTOM,
+	MAIN_MENU_CAMPAIGN_CHR,
+	MAIN_MENU_CAMPAIGN_HOTA,
+	MAIN_MENU_CAMPAIGN_WOG,
+	MAIN_MENU_CAMPAIGN_VCMI,
```

Comment:
Do we have any plans for those? Probably unnecessary at this point. 
And if we do get those, they might get called under different name.

Reply:
I have already prepared HotA and WoG Campaigns screens for 1.7. WoG campaigns are just converted to VCMP and VMAPs needs to be rebuilded, to make them playable, they won't be 1:1, but will be great to see them in game. 
And I have prepared main menu mod which contains all of these buttons when certain mods are enabled.
VCMI campaigns are currently not there, but there is plan to merge Burning Antagarich maps to first VCMI campaign as "Rise of Kastore" or something like that.

Reply:
Then once we get Kastore / Antagarich Burning I'd rather name them as such, and not as "VCMI".

And at the moment there is still just one short campaign, not entire campaign set.

Reply:
There is already, VCMI campaign button. Which I was thinking could be used for entering that screen.

https://github.com/vcmi-mods/h3-themes/blob/vcmi-1.6/h3-themes/mods/_Support/mods/OriginalButtons/content/sprites/images/vcmi-hover.png